adobe/react-spectrum
# RSP-1438 [Accessibility] fix keyboard navigation in useSelectableCollection

FocusRing should `continuePropagation` so that ancestors can handle focus/blur events.


Closes [RSP-1438](https://jira.corp.adobe.com/browse/RSP-1438)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding Issue [RSP-1438](https://jira.corp.adobe.com/browse/RSP-1438).

## 📝 Test Instructions:

1. Open http://localhost:9004/?path=/story/tree--long-list
2. Tab to set focus to the first item in the CollectionView list.
3. Use ArrowDown/ArrowUp/PageUp/PageDown/Home/End to move focus within the CollectionView list.

The focus should move to the appropriate item in the list and the focus-ring style should render.

## 🧢 Your Team:

Accessibility
